EIPS/eip-5484.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 5484
+title: Consensual Soulbound Tokens
+description: Interface for special NFTS with immutable ownership and pre-determined immutable burn authorization
+author: Buzz Cai (@buzzcai)
+discussions-to: https://ethereum-magicians.org/t/eip-5484-consensual-soulbound-tokens/10424
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-8-17
+requires: 165, 721
+---
+
+
+## Abstract
+This EIP proposes another soulbound token standard. The interface is an extension of [EIP-721](./eip-721.md). Before issuance, both parties (the issuer and the receiver), have to agree on who has the authorization to burn this token. Burn authorization is immutable after declaration. After its issuance, soulbound tokens can't be transferred, but can be burned based on a predetermined immutable burn authorization.  
+
+## Motivation
+The idea of soulbound tokens has gathered significant attention since its publishing. Without a standard interface, however, soulbound token developments are at a discord. It is hard to develop universal services targeting at soulbound tokens without minimal consensus on the implementation of the tokens. 
+
+This EIP envisions soulbound tokens as specialized NFTs that will play the roles of credentials, credit records, loan histories, memberships, and many more. In order to provide the flexibility in these scenarios, soulbound tokens must have an application-specific burn authorization and a way to distinguish themselves from regular EIP-721 tokens.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+- The token MUST implement the following interfaces:
+
+  1. [EIP-165](./eip-165.md)’s `ERC165` (`0x01ffc9a7`)
+  1. [EIP-721](./eip-721.md)’s `ERC721` (`0x80ac58cd`)
+
+- `burnAuth` SHALL be presented to receiver before issuance. 
+- `burnAuth` SHALL be Immutable after issuance.
+- `burnAuth` SHALL be the sole factor that determines which party has the rights to burn token.
+- The issuer SHALL present token metadata to the receiver and acquire receiver's signature before issuance.
+- The issuer SHALL NOT change metadata after issuance.
+
+### Contract Interface
+
```

Comment:
It's standard practice for EIP's using EIP-165 to include their own interface id explicitly.